ocs_ci/ocs/constants.py

```diff
@@ -603,7 +603,8 @@
 # Platforms
 AWS_PLATFORM = 'aws'
 VSPHERE_PLATFORM = 'vsphere'
-CLOUD_PLATFORMS = (AWS_PLATFORM)
+AZURE_PLATFORM = 'azure'
+CLOUD_PLATFORMS = (AWS_PLATFORM, AZURE_PLATFORM)
```

Comment:
I think it's better to change this tuple to a list

Reply:
Fixed !